modules/web/client/src/main/scala/observe/web/client/ObserveApp.scala

```diff
@@ -28,7 +27,7 @@ final class ObserveLauncher[F[_]](implicit val F: Sync[F], L: LiftIO[F]) {
   def serverSite: F[Site] =
     L.liftIO(IO.fromFuture {
       IO {
-        import ExecutionContext.Implicits.global
+        import org.scalajs.macrotaskexecutor.MacrotaskExecutor.Implicits._
```

Comment:
Cool, this should enable us to use scala.js 1.8

Reply:
Oh, this PR upgraded you already 😆 I didn't realize